packages/shared/lib/contexts/governance/utils/createProposalFromEvent.ts

```diff
@@ -1,22 +1,13 @@
 import { get } from 'svelte/store'
-
-import { ParticipationEventWithNodes } from '@iota/wallet'
-
+import type { ParticipationEventWithNodes } from '@iota/wallet/out/types'
 import { ProposalStatus, ProposalType } from '@contexts/governance/enums'
 import { IProposal } from '@contexts/governance/interfaces'
-import { nodeInfo, OFFICIAL_NODE_URLS } from '@core/network'
+import { OFFICIAL_NODE_URLS } from '@core/network'
 import { activeProfile, activeProfileId } from '@core/profile'
-import { getVotingEvents } from '../actions'
+import { getLatestProposalStatus, getParticipationsForProposal } from '..'
```

Comment:
For imports in the same folder, we've been adding explicit lines for each one, i.e.
```
import { getLatestProposalStatus } from './getLatestProposalStatus'
import { getParticipationsForProposal } from './getParticipationsFromProposal'
```